.gitignore

```diff
@@ -0,0 +1,17 @@
+.project
+.cproject
+.pydevproject
+*.pyc
+*.tar*
+*.orig
+bin
+lib
+build
+docs
+doc
+srv_gen
+msg_gen
+cmake_install.cmake
+*.cfgc
+*/cfg/cpp
+CATKIN_IGNORE
```

Comment:
feel free to use our default: https://github.com/ipa320/care-o-bot/blob/indigo_dev/.gitignore